packages/runner/package.json

```diff
@@ -24,6 +24,7 @@
     "@packages/rewriter": "0.0.0-development",
```

Comment:
Need to add in the test script to run the mocha tests  that were in runner-shared

Reply:
Currently, `runner-shared` unit tests are ignored. 

https://github.com/cypress-io/cypress/blob/f69afb5aa49996cace863f8a3ae142698deff982/package.json#L53

And it fails in the current `develop` branch.

And the only the file that exists is `studio-recorder.spec.js`. It's there for reference purpose and it'll be removed soon with #22870.

----

I checked the code base and removed `network`, `socket`, and `rewriter`.